backend/src/apiserver/client_manager.go

```diff
@@ -63,6 +67,7 @@ type ClientManager struct {
 	dBStatusStore          storage.DBStatusStoreInterface
 	defaultExperimentStore storage.DefaultExperimentStoreInterface
 	objectStore            storage.ObjectStoreInterface
+	objectPaths        storage.ObjectPathsInterface
```

Comment:
creating an interface for customizing the path seems a bit overkilling. 
can we add a root path in MinioObjectStore and set the path when initializing it? When add/get file we just append root path to the filepath?